fetchy/extractors/vaccinationDataPerDhbExtractor.ts

```diff
@@ -0,0 +1,27 @@
+import { DateTime } from "luxon";
+import { DhbVaccineDoseData } from "../../shared/types/VaccineDataTypes";
+import { extractDhbData_Including_and_Before_15_02_22 } from "./vaccinationDataPerDhbExtractors/extractorIncludingAndBefore15_02_22";
+import { extractDhbData_Including_and_Before_16_02_22 } from "./vaccinationDataPerDhbExtractors/extractorIncludingAndBefore16_02_22";
+import { extractDhbData_Including_and_After_17_02_22 } from "./vaccinationDataPerDhbExtractors/extractorIncludingAndAfter17_02_22";
```

Comment:
:minor: Since we'll have to rename this if / when the regex changes again, I think we should call it something a bit more generic like 'current'. Then cycle through renaming it if needed.

```suggestion
import { extractDhbDataCurrent } from "./vaccinationDataPerDhbExtractors/extractorCurrent";
```